packages/atlas/src/views/studio/CrtWelcomeView/CrtWelcomeView.tsx

```diff
@@ -1,26 +1,28 @@
-import { CrtView } from '@/views/studio/CrtView'
+import { atlasConfig } from '@/config'
+import { CrtMaintenanceView } from '@/views/studio/CrtMaintenanceView'
 
 export const CrtWelcomeView = () => {
-  return (
-    <CrtView />
```

Comment:
Why is this removed ?

Reply:
Mb, PR should target CRT branch. That way we no longer need a coming soon view which is what `crtView` was. Users will either see a landing page to create token or dashboard